### PR TITLE
Feature/group plug for controllers

### PR DESCRIPTION
**Pull request by hez**

### CODE REVIEW

File: lib/pxu_auth0/user_helpers.ex [hunk 1]
```diff
@@ -3,10 +3,29 @@ defmodule PxUAuth0.UserHelpers do
...
+  """
+  @spec in_group?(map, String.t() | list) :: boolean
+  def in_group?(conn, group_name) when is_binary(group_name),
     do: conn |> current_user() |> Map.get(:groups, []) |> Enum.member?(group_name)
+
+  def in_group?(conn, groups) when is_list(groups) do
+    conn
+    |> current_user()
+    |> Map.get(:groups, [])
+    |> Enum.any?(fn x -> Enum.member?(groups, x) end)
```
Comment (freemer) [inline lib/pxu_auth0/user_helpers.ex]: Heh, kinda surprised you didn't use the `&` shorthand here.

Comment (hez) [reply lib/pxu_auth0/user_helpers.ex]: I still find those hard to read sometimes

`Enum.any?(fn x -> Enum.member?(groups, x) end)`
vs
`Enum.any?(&Enum.member?(groups, &1))`
whats your pref?

Comment (freemer) [reply lib/pxu_auth0/user_helpers.ex]: Yeah, I go back and forth. The longhand is definitely more readable. Should we use prefer the longhand, unless the shorthand will keep a piped function to a single line?

Comment (hez) [reply lib/pxu_auth0/user_helpers.ex]: Feels like a place we can use our judgement, if it obscures it and makes it harder to read then use the other?

Comment (freemer) [reply lib/pxu_auth0/user_helpers.ex]: 👍

**Pull request merged**